src/Mono.Android/Test/Java.Interop-Tests/Java.Interop-Tests.NET.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project>
```

Comment:
I used the suffix `DotNet` on two projects already:

* https://github.com/xamarin/xamarin-android/blob/master/tests/CodeBehind/BuildTests/CodeBehindBuildTestsDotNet.csproj
* https://github.com/xamarin/xamarin-android/blob/master/tests/CodeBehind/CommonSampleLibrary/CommonSampleLibraryDotNet.csproj

But maybe `.NET` is better, do you want to also rename these two files?

Reply:
I would go with `.NET` now that I think about it. Since it will match the case of `Sdk="Microsoft.NET.Sdk"`.